### PR TITLE
Fix some material design and appcompat issues

### LINKED ISSUE #661: Crashes when pressing Status-Icons and Debugging
Hi @larsgrefer, I've just played a bit with our fresher app after you polished our design a bit with #654.
Looks and feels great so far, but our app now crashes on the following events (which it didn't do before):

* Pressing the Status Icons in the `About` menu
* Pressing `Debugging` in the Navigation Drawer

I've crafted a [logcat](https://defuse.ca/b/ByxKkHqk) for you, hopefully it contains enough information to fix what is wrong. Thank you!

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/28909526-crashes-when-pressing-status-icons-and-debugging?utm_campaign=plugin&utm_content=tracker%2F565750&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F565750&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by larsgrefer**
fix #661 
- use material themed `AlertDialog`s
- extend all Activities from AppCompatActivity
- correctly use `showAsAction` in menu resources

### CODE REVIEW

**Comment by larsgrefer** [comment]
Don not merge, its not finished

**Comment by larsgrefer** [comment]
@SecUpwN please review

File: app/src/main/java/com/SecUpwN/AIMSICD/fragments/AboutFragment.java [hunk 1]
```diff
@@ -173,4 +175,77 @@ public void onAttach(Activity activity) {
...
+
+    private void showInfoDialog(Status status) {
+        new AlertDialog.Builder(getActivity())
+                .setIcon(status.getIcon())
+                .setTitle(getString(R.string.status) + "\t" + getString(status.getName()))
+                .setMessage(status.getDescription())
+                .show();
+    }
+
+    public enum Status {
```
Comment (smarek) [inline app/src/main/java/com/SecUpwN/AIMSICD/fragments/AboutFragment.java]: Proper nice :+1:

**Comment by smarek** [comment]
I'm giving this :+1: reviewed

**Comment by banjaxbanjo** [comment]
Great job lovely coding there and a lot more easier to understand what is going on.

**Comment by larsgrefer** [comment]
@banjaxbanjo Self-documenting code :+1:

**Comment by larsgrefer** [comment]
@SecUpwN We have two positive reviews now, so I will merge this now

**Comment by SecUpwN** [comment]
>We have two positive reviews, so I will merge this now

Thanks for taking the lead while I was offline. Awesome work! :christmas_tree:

**Pull request merged**